NDB.Covid19/NDB.Covid19/ViewModels/MessagesViewModel.cs

```diff
@@ -11,9 +11,11 @@ public class MessagesViewModel
     {
         public static string MESSAGES_HEADER => "MESSAGES_HEADER".Translate();
         public static string MESSAGES_NO_ITEMS_TITLE => "MESSAGES_NOMESSAGES_HEADER".Translate();
+        public static string MESSAGES_NEW_MESSAGES_HEADER => "MESSAGES_NEW_MESSAGES_HEADER".Translate();
         public static string MESSAGES_NO_ITEMS_DESCRIPTION => "MESSAGES_NOMESSAGES_LABEL".Translate();
         public static string MESSAGES_LAST_UPDATED_LABEL => "MESSAGES_LAST_UPDATED_LABEL".Translate();
         public static string MESSAGES_ACCESSIBILITY_CLOSE_BUTTON => "MESSAGES_ACCESSIBILITY_CLOSE_BUTTON".Translate();
+        public static string MESSAGES_NEW_ITEM => "MESSAGES_NEW_ITEM".Translate();
```

Comment:
can you group this one and MESSAGES_NEW_MESSAGES_HEADER? so there is some structure e.g.,
MESSAGES_NEW_MESSAGES_HEADER
MESSAGES_NEW_ITEM
MESSAGES_NO_ITEMS_TITLE
MESSAGES_NO_ITEMS_DESCRIPTION

Reply:
done